pipeline/backend/kubernetes/kubernetes.go

```diff
@@ -353,11 +348,6 @@ func (e *kube) TailStep(ctx context.Context, step *types.Step, taskUUID string)
 }
 
 func (e *kube) DestroyStep(ctx context.Context, step *types.Step, taskUUID string) error {
-	if step.Type == types.StepTypeService {
-		// a service should be stopped by DestroyWorkflow so we can ignore it
-		log.Trace().Msgf("destroyStep got service '%s', ignoring it.", step.Name)
-		return nil
-	}
 	log.Trace().Str("taskUUID", taskUUID).Msgf("Stopping step: %s", step.Name)
 	err := stopPod(ctx, e, step, defaultDeleteOptions)
```

Comment:
in that case the `stopPod` or `startPod` should use `stepToPodName(step)` to gen the pod name